apps/ledger-live-mobile/src/screens/AddAccounts/01-SelectCrypto.tsx

```diff
@@ -94,7 +94,7 @@ export default function AddAccountsSelectCrypto({ navigation, route }: Props) {
       stargaze,
       umee,
       desmos,
-      secretNetwork,
+      secret_network: secretNetwork,
```

Comment:
it seems that we have just suffered from poor string typing here, could would add this?
it seems not easy to provide a non reg test, but adding a type will be that way to non regress this.
<img width="556" alt="Screenshot 2023-08-10 at 10 59 18" src="https://github.com/LedgerHQ/ledger-live/assets/211411/88af7839-cf25-4627-a7ef-d85e88dc10ab">

Reply:
by doing so, you can also remove the 2 comments that become useless since we prove by typing this "pay attention to the case" 👍

Reply:
as side note, it could also be a good idea to move this logic out of the UI side, 01-SelectCrypto.tsx is a generic screen, what if another screen need this, 
not even talking about the non modular aspect of this, but at least this should be moved in a hooks/ helper maybe? 
`function useFeatureFlaggedCurrencies`